docs/config.md

```diff
@@ -0,0 +1,89 @@
+# The InSpec Configuration File
+
+This documents the InSpec configuration file format introduced in version 3.5 of InSpec.
+
+## Config File Location
+
+By default, InSpec looks for a config file in `~/.inspec/config.json`.  InSpec does not need a configuration file to run.
+
+You may also specify the location using `--config`.  For example, to run the shell using a config file in `/etc/inspec`, use `inspec shell --config /etc/inspec/config.json`.
+
+## Config File Format Versions
+
+Config files must contain a top-level key, `version`, which indicates the file format. This allows us to add new fields without breaking old installations.
+
+## Version 1.1
+
+### Complete Example
+
+```
+{
+  "version": "1.1",
+  "cli_options":{
+    "color": "true"
+  },
+  "credentials": {
+    "ssh": {
+      "my-target": {
+        "host":"somewhere.there.com",
+        "user":"bob"
+      }
+    }
+  },
+  "reporter": {
+    "automate" : {
+    "stdout" : false,
+    "url" : "https://YOUR_A2_URL/data-collector/v0/",
+    "token" : "YOUR_A2_API_TOKEN",
+    "insecure" : true,
+    "node_name" : "inspec_test_node",
+    "environment" : "prod"
+  }
+}
+```
+
+
+### version
```

Comment:
Let's use a definition list for these:

```
version
: Should have the value '1.1'
```

Reply:
Seconded on the definition list suggestion.